src/aat/java/uk/gov/hmcts/reform/coh/functional/bdd/steps/ApiSteps.java

```diff
@@ -231,4 +236,13 @@ public void sendGetRequestToTheLocation() {
         testContext.getHttpContext().setHttpResponseStatusCode(response.getStatusCodeValue());
         testContext.getHttpContext().setRawResponseString(response.getBody());
     }
+
+    @And("^an event has been queued for this online hearing of event type (.*)$")
+    public void anEventHasBeenQueuedForThisOnlineHearingOfEventType(String eventType) throws Throwable {
+        OnlineHearing onlineHearing = testContext.getScenarioContext().getCurrentOnlineHearing();
+        Optional<SessionEvent> optSessionEvent = sessionEventService.retrieveByOnlineHearing(onlineHearing);
```

Comment:
Could there be more than 1 sessionEvent per online hearing?

Reply:
Good point. I'll fix that. Actually I see that you've implemented a findAll... and I steal that off your branch :-)